spec/hydra/works/services/generic_file/upload_file_spec.rb

```diff
@@ -2,24 +2,24 @@
 
 describe Hydra::Works::UploadFileToGenericFile do
 
-  let(:generic_work)        { Hydra::Works::GenericWork::Base.new }
-  let(:generic_file)        { Hydra::Works::GenericFile::Base.new }
-  let(:filename)            { "sample-file.pdf" }
-  let(:file)                { File.join(fixture_path, filename) }
-  let(:updated_filename)    { "updated-file.txt"}
-  let(:updated_file)        { File.join(fixture_path, updated_filename) }
+  let(:generic_work)        { Hydra::Works::GenericWork::Base.create }
```

Comment:
Can you use `#new` instead of `#create` on this line and the next line (to improve test performance)?